EXTENSIONS_CSS_MAP

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
please revert this file

Reply:
https://github.com/ampproject/amphtml/pull/27667/commits/b4be3c1635e05c0aed663953fe4829a8bdd45767